src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -219,7 +223,7 @@ const StoragePoolForm: FC<Props> = ({ formik, section, setSection }) => {
               readOnly={formik.values.readOnly}
             >
               <Notification severity="information" title="YAML Configuration">
-                This is the YAML representation of the storage pool.
+                {`${!supportedStorageDrivers.has(formik.values.driver) ? `The ${formik.values.driver} driver is not fully supported in the web interface. ` : ""}This is the YAML representation of the storage pool.`}
```

Comment:
Thought it would be good to add some info here indicating that the driver for the storage pool is not fully supported in the UI. Open to suggestions for better copy :slightly_smiling_face: